src/olcs/contrib/manager.js

```diff
@@ -229,10 +234,12 @@ olcs.contrib.Manager = class {
         goog.asserts.assert(this.map);
         return olcs.core.resetToNorthZenith(this.map, scene).then(() => {
           ol3d.setEnabled(false);
+          this.dispatchEvent(new ol.events.Event(ol.events.EventType.CHANGE));
```

Comment:
Better call the event `toggle`. Then it is fine to have it on the `manager` instead of `ol3d`.

Reply:
What do you mean ? `ol.events.EventType.CHANGE` to be `'toggle'` ? So i need to create a new type ? It's not in ol event types list.

Also i saw a function `changed` in `ol.Observable` that i would use..